algorithms/gr-bfsrh/javascript/bfs.js

```diff
@@ -0,0 +1,47 @@
+const bfs = (graph, root, targetNode) => {
```

Comment:
Codacy found an issue: [Arrow function has a complexity of 7. Maximum allowed is 5.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6319059)